airwallex/src/main/java/com/airwallex/android/AirwallexStarter.kt

```diff
@@ -135,12 +136,14 @@ class AirwallexStarter {
         fun presentPaymentFlow(
             fragment: Fragment,
             session: AirwallexSession,
-            paymentResultListener: Airwallex.PaymentResultListener
+            layoutType: PaymentMethodsLayoutType,
```

Comment:
We can give it a default value = .TAB (for other public methods here too) so it's easier to use. Also, could you please help add the @param comment, thanks.